add tips for CEL expression (#14876)
istio/istio.io: content/en/docs/tasks/observability/logs/telemetry-api/index.md

    EOF
    {{< /text >}}

    {{< tip >}}

>     There's no `response.code` attribute at all when connection fails, in that case, you should use CEL expression `!has(response.code) || response.code >= 500`.

Reviewer: ```suggestion
    There's no `response.code` attribute when connections fail. In that case, you should use the CEL expression `!has(response.code) || response.code >= 500`.
```
